tests/playwright-test/global-setup.spec.ts

```diff
@@ -21,7 +21,7 @@ test('globalSetup and globalTeardown should work', async ({ runInlineTest }) =>
     'playwright.config.ts': `
       import * as path from 'path';
       module.exports = {
-        globalSetup: 'globalSetup.ts',
+        globalSetup: './globalSetup',
```

Comment:
Since we preserve the old mode, let's leave a test that uses it without a relative path.

Reply:
there are some down below that i left unchanged.